### PR TITLE
Gitops bridge v2

### PR DESCRIPTION
**Pull request by csantanapr**
Support for new ArgoCD gitops bridge

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: argocd/bootstrap/control-plane/addons/aws/addons-aws-oss-karpenter-appset.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  generators:
+    - merge:
+        mergeKeys:
+          - server
+        generators:
+          - clusters:
+              values:
+                addonChart: karpenter
+                # anything not staging or prod use this version
+                addonChartVersion: 'v0.27.2'
```
Comment (FernandoMiguel) [inline argocd/bootstrap/control-plane/addons/aws/addons-aws-oss-karpenter-appset.yaml:17]: [v0.29.2](https://github.com/aws/karpenter/releases/tag/v0.29.2)

File: argocd/charts/addons/argo-cd/values.yaml [hunk 2]
```diff
@@ -0,0 +1,218 @@
+
+
```
Comment (FernandoMiguel) [inline argocd/charts/addons/argo-cd/values.yaml:2]: empty lines

**Review by allamand — APPROVED** [review]
LGTM. I've tested it with 2 differents configurations (route53 domain name and external nginx)

**Comment by csantanapr** [comment]
Thank @allamand for the review and all the feedback and contributions 🎉

**Pull request merged**